includes/controls/groups/grid-container.php

```diff
@@ -295,6 +296,17 @@ protected function get_responsive_unit_defaults() {
 		return $responsive_unit_defaults;
 	}
 
+	protected function get_responsive_autoflow_defaults() {
+		$responsive_autoflow_defaults = [];
+		$active_breakpoints = Plugin::$instance->breakpoints->get_active_breakpoints();
+
+		foreach ( $active_breakpoints as $breakpoint_name => $breakpoint ) {
+			$responsive_autoflow_defaults[ $breakpoint_name . '_default' ] = 'row';
+		}
+
+		return $responsive_autoflow_defaults;
+	}
```

Comment:
@mykytamurzin Why did you need to do this? In the JS you can use `elementorFrontend.utils.controls.getResponsiveControlValue()`. This will get the correct value, if the smaller device is empty it will inherit the value from the bigger device.

Reply:
These default values are checking in PHPUnit tests, they were failing without it. See 
https://github.com/elementor/elementor/actions/runs/5781501240/job/15666703826#step:7:177

Reply:
I don't need them for my feature, but after adding frontend_available=true - they started popping up